src/main/resources/xslt/outputs/odt/models.xsl

```diff
@@ -1,544 +1,641 @@
+				<!--<text:p text:style-name="Instruction"><xsl:value-of select="$instructionFormatMaj"/></text:p>-->
+			</xsl:when>
+			<xsl:when test="$instructionFormat='warning'">
+				<text:p text:style-name="Warning"><xsl:value-of select="$instructionLabel"/></text:p>
+				<!--<text:p text:style-name="Warning"><xsl:value-of select="$instructionFormatMaj"/></text:p>-->
+			</xsl:when>
+			
+		</xsl:choose>
+		
+		<!-- Go to the children -->
+		<xsl:apply-templates select="eno:child-fields($source-context)" mode="source">
+			<xsl:with-param name="driver" select="." tunnel="yes"/>
+		</xsl:apply-templates>
+		
+	</xsl:template>
+
+	<xsl:template match="CalculatedVariable" mode="model">
+		<xsl:param name="source-context" as="item()" tunnel="yes"/>
+		<xsl:variable name="languages" select="enoodt:get-form-languages($source-context)" as="xs:string +"/>
+		<xsl:variable name="variableCalculation" select="enoodt:get-calculate($source-context)"/>
+		<xsl:variable name="variableCalculationLabel" select="enoodt:get-calculate-text($source-context,$languages[1],'label')"/>
+		<xsl:variable name="variableCalculationAlert" select="enoodt:get-calculate-text($source-context,$languages[1],'alert')"/>
+		
+		<text:p><xsl:value-of select="$variableCalculation"/></text:p>
+		<text:p><xsl:value-of select="$variableCalculationLabel"/></text:p>
+		<text:p><xsl:value-of select="$variableCalculationAlert"/></text:p>
+		
+		<!-- Go to the children -->
+		<xsl:apply-templates select="eno:child-fields($source-context)" mode="source">
+			<xsl:with-param name="driver" select="." tunnel="yes"/>
+		</xsl:apply-templates>		
+	</xsl:template>
+	
+	<xsl:template match="Control" mode="model">
+		<xsl:param name="source-context" as="item()" tunnel="yes"/>
+		<xsl:variable name="languages" select="enoodt:get-form-languages($source-context)" as="xs:string +"/>
+		<xsl:variable name="name" select="enoodt:get-label-conditioner($source-context,$languages[1])"/>
+		<xsl:variable name="nameOfControl" select="enoodt:get-label($source-context,$languages)"/>
+		<xsl:variable name="control" select="enoodt:get-constraint($source-context)"/>
+		
+		<xsl:variable name="items">
+			<xsl:call-template name="splitStringToItems">
+				<xsl:with-param name="delimiter" />
+				<xsl:with-param name="list" select="$control" />
+			</xsl:call-template>
+		</xsl:variable>
+		
+		<xsl:for-each select="$items">
+			<text:p text:style-name="Control">
+				<xsl:value-of select="."/>
+			</text:p>
+		</xsl:for-each>
+		
+		<!-- Go to the children -->
+		<xsl:apply-templates select="eno:child-fields($source-context)" mode="source">
+			<xsl:with-param name="driver" select="." tunnel="yes"/>
+		</xsl:apply-templates>		
+	</xsl:template>
+		
+		
+	<xsl:template name="splitStringToItems">
+		<xsl:param name="list" />
+		<xsl:param name="delimiter" select="'+'"  />
+		<xsl:variable name="_delimiter">
+			<xsl:choose>
+				<xsl:when test="string-length($delimiter)=0">,</xsl:when>

```

Comment:
If I understand well, when no value is passed, the '+' is used as a default value for the params, but if it's the empty string it's the ',' which is used as the default value ? Is this the implementation really expected ?

I didn't deeply check the implementation, but take a look at the tokenize() xpath function, could help ;-)